bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/DigitalInvoiceScreenTests.kt

```diff
@@ -172,27 +169,25 @@ class DigitalInvoiceScreenTests {
         captureScreen.clickFiles()
         pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
         idlingResource.waitForIdle()
-        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.checkDigitalInvoiceButtonOnOnboardingScreenIsDisplayed()
         digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
         idlingResource.waitForIdle()
-        digitalInvoiceScreen.assertDigitalInvoiceText()
         digitalInvoiceScreen.clickHelpButtonOnDigitalInvoiceScreen()
         digitalInvoiceScreen.verifyHelpTextOnNextScreen()
         digitalInvoiceScreen.verifyFirstTitleOnHelpScreen()
```

Comment:
[Request] Same comment as above - please add an assertion at the end.

Reply:
Assertion added.

```
val isHelpTextVisible = digitalInvoiceScreen.verifyHelpTextOnNextScreen()
assertEquals(true, isHelpTextVisible)

val isFirstTitleTextVisible = digitalInvoiceScreen.verifyFirstTitleOnHelpScreen()
assertEquals(true, isFirstTitleTextVisible)
```